corehq/apps/users/models.py

```diff
@@ -1088,13 +1088,19 @@ def sync_to_django_user(self):
             django_user = User(username=self.username)
         for attr in DjangoUserMixin.ATTRS:
             attr_val = getattr(self, attr)
-            if not attr_val and attr != 'last_login':
+            if attr in [
+                'is_active',
+                'is_staff',
+                'is_superuser',
+            ]:
+                attr_val = bool(attr_val)
```

Comment:
i see. to be safe it might be best to say:

`attr_val = attr_val if attr_val is True else False`

it just seems a bit dangerous to set `is_superuser` to the outcome of `bool(attr_val)`, in case `attr_val` gets set to something like `'False'`